david_vicklund/gulpfile.js

```diff
@@ -23,6 +28,29 @@ gulp.task('jshint:test', function() {
   .pipe(jshint.reporter('jshint-stylish'));
 });
 
+gulp.task('webpack:test', function() {
+  return gulp.src('test/client/test_entry.js')
+    .pipe(webpack({
+      output: { 
+        filename: 'test_bundle.js'
+      }
+    }))
+    .pipe(gulp.dest('test/client/'));
+});
+
+// CSS tasks
+gulp.task('minify:scss', function() {
+  gulp.src('app/scss/application.scss')
+    .pipe(sass())
+    .pipe(gulp.dest('app/css/'));
+  return gulp.src('app/css/**/*.css')
+    .pipe(sourcemaps.init())
+    .pipe(concatCss('styles.min.css'))
+    .pipe(minifyCss())
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest('build/'));
+});
+
```

Comment:
something looks messed up here. Fix this up and get more points. All teh other CSS stuff looks good.